### PR TITLE
Opensearch 2.12 - terraform registry documentation update

### PR DESCRIPTION
**Pull request by arjunrajnetapp**
Internal reference: INS-27733

### CODE REVIEW

**Review by iclonghoang — APPROVED** [review]

**Comment by tan3-netapp** [comment]
It's interesting that we have added `gitleak` here. I didn't add in my change. LGTM, though.

**Review by chengwushi-netapp — APPROVED** [review]
lgtm

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 BIN_NAME=terraform-provider-instaclustr
 
-VERSION=2.0.101
+VERSION=2.0.102
```
Comment (isuru-instaclustr) [inline Makefile:3]: need to update to correct version. Also best to sync with the latest master.

**Comment by fahadshamiinsta** [comment]
moved changes to https://github.com/instaclustr/terraform-provider-instaclustr/pull/380. Closing this PR

**Pull request closed without merging**